protocol/p2p/src/core/router.rs

```diff
@@ -327,6 +384,11 @@ impl Router {
         }
     }
 
+    pub async fn enqueue_from(&self, mut msg: KaspadMessage, id: u32) -> Result<(), ProtocolError> {
```

Comment:
should be removed and instead callers should use `make_message!` with the `id` overload